calculator.rb

```diff
@@ -0,0 +1,64 @@
+# Calculator Project
+
+# Collect user input for first number
+puts "Welcome to calculator! Please enter the first number you'd like to use:"
+num_one = gets.chomp
+# Checks to see if user input is a number. Alerts user of incorrect input
```

Comment:
I might put in a comment here to explain why this logic works to detect incorrect input, since it's a little tricky.